### DISCUSSION TITLE
Error: Type is not assignable to type 'never'. Typescript

### DISCUSSION THREAD

**Discussion by kpguru20001** [prisma/prisma#14031] [Q&A] [answered]
Particular Structure of Database structure doesn't allow a nested write.
`Type 'void' is not assignable to type 'never'.ts(2322)`

Prisma Schema:
```
model TestSKU {
  id           Int    @id @default(autoincrement())
  name     String

  TestCompSKURel TestUseSKURelation[]
}

model TestUseSKURelation {
  id                  Int        @id @default(autoincrement())
  name            String
  testA            TestSKU    @relation(fields: [testAId], references: [id])
  testAId          Int
  testC             TestSKUUse @relation(fields: [testCId], references: [id])
  testCId          Int

  @@unique([testAId, testCId])
}

model TestSKUUse {
  id                              Int              @id @default(autoincrement())
  name                        String
  TestCompSKURel   TestCompSKURel[]
  TestD                       TestComp         @relation(fields: [testDId], references: [id])
  testDId                     Int
}

model TestComp {
  id          Int    @id @default(autoincrement())
  name    String

  TestC   TestSKUUse[]
}
```

Here this is the error I am getting:
<img width="534" alt="image" src="https://user-images.githubusercontent.com/32976334/176149413-ac7afbb4-e698-4b50-bc42-2fe20c75beac.png">

**Comment by austincrim** [comment, marked as answer]
Hey @kpguru20001 👋,

I'm seeing the same thing on my side, seems kind of unexpected. Would you mind opening a [bug report](https://github.com/prisma/prisma/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.yml)?

You can also use the `createMany` option which looks a bit different but should accomplish the same thing:
```ts
await prisma.testSKU.create({
  data: {
    name: 'foo',
    TestCompSKURel: {
      createMany: {
        data: [{ name: 'foo', testCId: 1 }]
      }
    }
  }
})
```

You can read about the difference here. Let me know if I can clarify anything else!

**Reply by kpguru20001** [reply]
Thankyou @austincrim, Will report a bug.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
